umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/distributed_format_util.clj

```diff
@@ -87,7 +93,7 @@
    string if ', ' exists or the input string if ', ' does not exist."
   [formats]
   (if (string/includes? formats ",")
-    (string/split formats #", *")
+    (remove-emtpy-strings (string/split formats #", *"))
```

Comment:
Nitpick: you could thread-first this to make it a little more readable
```suggestion
    (-> formats
        (string/split #", *") 
        remove-emtpy-strings)
```

Reply:
I have applied the changes.